scalafmt-intellij/src/main/scala/org/scalafmt/intellij/FileDocument.scala

```diff
@@ -17,13 +17,15 @@ case class FileDocument(project: Option[Project], document: Document) {
   def path = virtualFile.getPath
   def isSbt: Boolean =
     virtualFile.getFileType.getName.compareToIgnoreCase("SBT") == 0
+  def isSc: Boolean =
+    virtualFile.getFileType.getName.compareToIgnoreCase("SC") == 0
```

Comment:
Did you check the file type in IntelliJ for .sc files?

Reply:
Ah good call. I had no idea what I was doing in IntelliJ and just cargo-culted.

Apparently it's a Scala file, so I needed to check for the `.sc` extension.